web3/packages/api-server/src/base/gw-config.ts

```diff
@@ -237,11 +236,6 @@ export class GwConfig {
   private async fetchDefaultFromAccount(
```

Comment:
This comment `// Using zero address firstly` could be deleted?

Reply:
Updated